### DISCUSSION TITLE
Zigbee Repeater - Did it make it into the mainstream V2.2?

### DISCUSSION THREAD

**Discussion by rb0135** [seaky/nspanel_pro_tools_apk#136] [Q&A]
Hi All,

A bit confused....

Just asking if the Zigbee Repeater functionality it is now included in these main tools, or do we still need to add svanrossem scripts to make it work [https://gitlab.com/svanrossem/sonoff-nspanelpro-scripts](url)

Reason I ask is in the early days, I used the scripts and I was able to get the Zigbee repeater to work, but it keeps dropping out and the devices connected to it obviously drop out. A reboot of the Panel seems to bring them available for a little while, then they drop out after a certain time (not sure of the time they are connected for). My HA Zigbee Diagram shows the Panel Zigbee is dropping offline, not the zigbee devices.

So, the main question is if it is integrated, then Im presuming I need to remove the manual installed scripts from svanrossem?

I did read all the info, but could not determine from the release notes if it is actually incorporated yet,

Thanks all..

**Comment by seaky** [comment]
still working on it

**Comment by rb0135** [comment]
Hi @seaky - Thanks for replying... That's great, thanks...
